packages/ui/VirtualList/VirtualListBase.js

```diff
@@ -1058,21 +1058,26 @@ const VirtualListBaseFactory = (type) => {
 
 		// render
 
-		mergeClasses = (className) => {
+		makeContainerClasses (className) {
```

Comment:
Changed this and `getContentClasses` to prototype functions instead of member arrow functions. I tend to prefer prototype functions unless we are passing them around since member functions are transpiled to be created at construction which incurs unnecessary overhead.